website_sale_require_legal/static/src/js/tour.js

```diff
@@ -8,7 +8,23 @@
 
     var steps = [
         {
-            trigger: "a:contains('iPod')",
+            content: "select conference chair",
```

Comment:
:fire: You don't need the `content` key if it's a [tour-only test](https://github.com/OCA/e-commerce/pull/267/commits/c292c82a9fab4a6a9a95a44f2bf625b60433703f#diff-45d95ed91b5ba60838fc08079f91087aR65).